pkgs/development/python-modules/anyqt/default.nix

```
@@ -0,0 +1,23 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "AnyQt";
+  version = "0.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-S5IouRiibfL9zGHxNdtUitPDA25bx+Z7qBR+C2aDdjs=";
+  };
+
+  doCheck = false; # TODO: xvfb stuff
+
+  meta = {
+    description = "PyQt/PySide compatibility layer.";
```

```suggestion
    description = "PyQt/PySide compatibility layer";
```

Short description does not end with punctuation.